scripts/docs-deploy/clone-docs-repo.ts

```diff
@@ -0,0 +1,62 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import {$} from 'zx';
```

Comment:
This script is using `zx` while others are using `shelljs`. My understanding is that the two packages are similar so should we settle on one of them?

Reply:
Yes, I wanted to leave a PR comment for that. I want to slowly start using `zx` in the COMP repo as ShellJS requires the extra types package and Zx has various benefits, including some bugs fixed that make ShellJS hard to reliably use.

e.g. `shelljs.exec("yarn whatever-fails", {fatal: true})` does not actually work as expected (bug has been around for quite some time)